ci/infra/openstack/generate-cloud-provider-conf

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+if [ -z "${OS_AUTH_URL}" ] || [ -z "${OS_USERNAME}" ] || \
+   [ -z "${OS_PASSWORD}" ] || [ -z "${OS_PROJECT_ID}" ] || \
+   [ -z "${OS_PRIVATE_SUBNET_ID}" ] || [ -z "${OS_PUBLIC_NET_ID}" ]; then
+  echo '[ ERROR ] $OS_AUTH_URL $OS_USERNAME $OS_PASSWORD $OS_PROJECT_ID'
+  echo '[ ERROR ] $OS_PRIVATE_SUBNET_ID $OS_PUBLIC_NET_ID must be specified'
+  echo '[ ERROR ] Please download and source your OpenStack RC file'
```

Comment:
Errors should probably go to stderr, either `>&2` or, more readably, `>/dev/stderr`

Reply:
Sure, let's define some logging functions